data/mods/Magiclysm/itemgroups/spellbooks.json

```diff
@@ -76,7 +76,8 @@
       [ "spell_scroll_impactsling", 20 ],
       [ "spell_scroll_boneclub", 20 ],
       [ "spell_scroll_flamesword", 35 ],
-      [ "spell_scroll_flamebreath", 30 ]
+      [ "spell_scroll_flamebreath", 30 ],
+      [ "spell_scroll_summon_wolf", 30 ]
```

Comment:
Tier 1 or Tier 2, with summon_decayed_pouncer (Animist)? What are the helpful combat capabilities of a wolf vs a skeleton or decayed pouncer?

Reply:
Sorry are you saying i should lower the tier?

Reply:
Currently, it's down among the Tier 1 spells. Should it be that or Tier 2?

From the Animist summoning spells, what makes the difference on the tier seems to be the combat capabilities of the summoned being(s). Do wolves compare better to skeletons (Tier 1) or decayed pouncers (Tier 2)?

Reply:
your right i'm hoing to up the Tier by 1 i think i made 3 skeletons fight 3 wolfs 3 times and they lost every time without killing one wolf.

Reply:
OK. Upping the difficulty a bit may be advised, then - summon decayed pouncers (Tier 2) is difficulty 12. On the other hand, another Tier 2 spell, the druidic Feral Form, is only difficulty 4, so I'm not sure.

Reply:
Yeah i'm not sure maybe i will up it a little the main thing is i know there is an artic wolf summon and maybe i'm wrong but they have a diffculty of 7.

Reply:
Point. I don't know to what degree that's supposed to be counterbalanced by it only being available via an attunement - I missed those when doing a search for "summon". (BTW, @KorGgenT - are attunements supposed to be available with either of the two prerequisite classes, or only with both? Currently, since it's all in one PREREQ field, I believe it would require only one currently - PREREQ2 is used to get a second requirement.) I'll take a further look at that when I have some time.

Reply:
> 
> 
> Point. I don't know to what degree that's supposed to be counterbalanced by it only being available via an attunement - I missed those when doing a search for "summon". (BTW, @KorGgenT - are attunements supposed to be available with either of the two prerequisite classes, or only with both? Currently, since it's all in one PREREQ field, I believe it would require only one currently - PREREQ2 is used to get a second requirement.) I'll take a further look at that when I have some time.

Good point I will increase it to 6 maybe that will be a better fit.

Reply:
> Point. I don't know to what degree that's supposed to be counterbalanced by it only being available via an attunement - I missed those when doing a search for "summon". (BTW, @KorGgenT - are attunements supposed to be available with either of the two prerequisite classes, or only with both? Currently, since it's all in one PREREQ field, I believe it would require only one currently - PREREQ2 is used to get a second requirement.) I'll take a further look at that when I have some time.

you must have both. there's some extra code for getting attunements that uses both fields.